--- die_sourcecode_3.09/XOptions/xoptions.cpp	2023-12-28 18:09:21.000000000 -0300
+++ die_sourcecode_3.09/XOptions/xoptions.cpp	2024-02-09 22:18:09.427362084 -0300
@@ -1534,7 +1534,7 @@
     QString sApplicationDirPath = qApp->applicationDirPath();
 
     if ((sApplicationDirPath == "/bin") || (sApplicationDirPath == "/usr/bin") || (sApplicationDirPath == "/usr/local/bin") ||
-        (sApplicationDirPath.contains("/usr/local/bin$")) || isAppImage()) {
+        (sApplicationDirPath == "/app/bin") || (sApplicationDirPath.contains("/usr/local/bin$")) || isAppImage()) {
         bResult = true;
     } else {
         bResult = false;
